docs/console/docs/03-02-ui-api-layer.md

```diff
@@ -9,27 +9,27 @@ The UI API Layer is a backend service which provides an API for all views of the
 
 ## Cache
 
-For GraphQL queries, the UI API Layer uses caching which is based on Informers from the Kubernetes Go client. There are separate cache stores for every Kubernetes resource. The stores are synchronized when the service starts. After cache synchronization, a single connection with the Kubernetes API server is established and any event related to one of the observed resources updates the given cache store. This logic ensures that cache stores are always up-to-date without sending multiple requests to the Kubernetes API server.
+For GraphQL queries, the UI API Layer uses caching which is based on Informers from the Kubernetes Go client. There are separate cache stores for every Kubernetes resource. The stores are synchronized when the service starts. After cache synchronization, a single connection with the Kubernetes API server is established and any event related to one of the observed resources updates the corresponding cache store. This logic ensures that cache stores are always up-to-date without sending multiple requests to the Kubernetes API server.
 
 ## Modularization
 
 The UI API Layer consists of the Kubernetes resource logic and cache for different domains, such as the Service Catalog, Application, or Kubeless. The UI API Layer introduces modularization changes which are based on toggling modules while the server is running. The enabled module synchronizes cache for its resource and enables the module's logic for all server requests. If you disable a given module, every GraphQL query, mutation, and subscription related to this module returns an error.
 
 The UI API Layer module pluggability is hidden behind a feature toggle. It is not enabled by default because the Console UI still requires resiliency improvements to ensure no errors occur when a certain Kyma component is not installed.
```

Comment:
I wanted to rephrase this sentence as well, but I'm not sure how to do it. My draft was: 
`The UI API Layer module pluggability must be enabled through feature toggle. You may experience errors if the component for which you want to use the GraphQL module is not installed.`
Can you verify if this is okay?

Reply:
Actually, the whole paragraph can be deleted as there is no feature toggle anymore.

Reply:
cool, thanks :)